.github/workflows/build-sbom-utility-scripts-image.yml

```diff
@@ -1,80 +0,0 @@
-name: Build sbom utility scripts image
-
-on:
-  push:
-    branches:
-      - main
-    paths:
-      - .github/workflows/build-sbom-utility-scripts-image.yml
-      - sbom-utility-scripts/**
-
-  pull_request:
-    branches:
-      - main
-    paths:
-      - .github/workflows/build-sbom-utility-scripts-image.yml
-      - sbom-utility-scripts/**
-
-
-env:
-  REGISTRY: quay.io/redhat-appstudio
-  IMAGE_NAME: sbom-utility-scripts-image
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-    - name: Checkout repository
-      uses: actions/checkout@v4
-
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: 3.11
-
-    - name: Run tox checks for base-images-sbom-script
-      run: |
-        python3 -m pip install tox
-        cd ./sbom-utility-scripts/scripts/base-images-sbom-script/app/
-        tox
-
-    - name: Run tox checks for merge-cachi2-sboms-script
-      run: |
-        python3 -m pip install tox
-        cd ./sbom-utility-scripts/scripts/merge-cachi2-sboms-script/
-        tox
-
-    - name: Run tox checks for index-image-sbom-script
-      run: |
-        python3 -m pip install tox
-        cd ./sbom-utility-scripts/scripts/index-image-sbom-script/
-        tox
-
-    - name: Run tox checks for index-image-sbom-script
-      run: |
-        python3 -m pip install tox
-        cd ./sbom-utility-scripts/scripts/add-image-reference-script/
-        tox
```

Comment:
Maybe we should keep this part? I don't think we have a Konflux replacement for running the unit tests